gitignore

```diff
@@ -25,7 +25,8 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-# local env files
+# env files
+.env
```

Comment:
Just getting the .env file out of the repo. Doesn't have any actual security credentials—so, just a proactive step to prevent security leak. ;)